### PR TITLE
add simage recipe

### PR DESCRIPTION
**Pull request by looooo**
1. part of the pivy feedstock

### CODE REVIEW

File: recipes/simage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+source:
+    fn: {{ name }}.tar.gz
+    url: {{ download_url }}
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [py<35]
+    skip: true  # [osx]
```
Comment (bgruening) [inline recipes/simage/meta.yaml]: This is not supported afaik, plese use something like `# [py<35 or osx]` in a single line.

File: recipes/simage/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+requirements:
+    build:
+        - toolchain
+        - vc 14  # [win]
+        - ninja  # [win]
+        - gcc  # [linux]
+        - cmake
+        - qt 5.*
+        - libpng
+        - jpeg
```
Comment (bgruening) [inline recipes/simage/meta.yaml]: please pin libpng and jpeg as well according to this table: https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L64-L129

**Comment by looooo** [comment]
should be ready now.

File: recipes/simage/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+build:
+    number: 0
+    skip: true  # [py<35 or osx]
+
+    features:
+        - vc14  # [win]
+
+    track_features:
+        - vc14  # [win]
```
Comment (isuruf) [inline recipes/simage/meta.yaml]: Above 2 lines should be removed

File: recipes/simage/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+requirements:
+    build:
+        - toolchain
+        - vc 14  # [win]
```
Comment (isuruf) [inline recipes/simage/meta.yaml]: Add `- python # [win]` here.

File: recipes/simage/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    version: {{ version }}
+
+source:
+    fn: {{ name }}.tar.gz
+    url: {{ download_url }}
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [py<35 or osx]
```
Comment (isuruf) [inline recipes/simage/meta.yaml]: Change this to `[(win and py<35) or osx]`

File: recipes/simage/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,65 @@
+{% set name = "simage" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "4b57ed3c4997fbcc8b13823a60c1115cd97f701ada8afd44241f8512a5a9adc0" %}
+{% set download_url = "https://github.com/FreeCAD/pivy/releases/download/0.6.3/simage.tar.gz" %}
```
Comment (isuruf) [inline recipes/simage/meta.yaml]: Please use the official release for simage

Comment (looooo) [reply recipes/simage/meta.yaml]: there is no official release for the cmake-branch of simage.

Comment (isuruf) [reply recipes/simage/meta.yaml]: Can you use `https://bitbucket.org/Coin3D/simage/get/cf953eacd849.tar.gz`? It's the latest master. Also you can do it like
`{% set commit = "cf953eacd849" %}` and make the url `https://bitbucket.org/Coin3D/simage/get/{{ commit }}.tar.gz` below

**Comment by looooo** [comment]
@conda-forge/staged-recipes please merge.

**Comment by looooo** [comment]
thanks

**Pull request merged**